### DISCUSSION TITLE
why nativescript preview always opens the browser when reloading

### DISCUSSION THREAD

**Discussion by bobwatcherx** [NativeScript/NativeScript#10085] [Q&A]
is there a solution to speed up reloading in nativescript preview app, when reloading my app.  I feel annoyed and feel long while waiting for small changes

https://user-images.githubusercontent.com/52072811/201055212-064c8629-1854-4b82-ae8c-3ce2deb7ab59.mp4

**Comment by rigor789** [comment]
There's a pending issue on our side to fix instant reloads (HMR) that was accidentally broken in the last release, we'll have an update soon that should fix it for most cases, and only restart when necessary.
